server/src/main/java/org/elasticsearch/common/breaker/ChildMemoryCircuitBreaker.java

```diff
@@ -208,15 +185,15 @@ public long getUsed() {
      */
     @Override
     public long getLimit() {
```

Comment:
It looks like the one use of this is to grab stats for a breaker. Due to this and overhead being separate methods, this could theoretically result in a race condition where we get incorrect configuration in the stats. I realize this is already currently an issue, so this doesn't need to be solved right now, but could you please create a followup issue to solve this, maybe by getting rid of these methods and having a stats() method directly on the breaker.

Reply:
@rjernst definitely, I will create a new ES issue. It does not make sense to have separate methods any longer.